tests/generation/FileGlobs.py

```diff
@@ -56,14 +56,34 @@ def test_get_project_files_gitignore(self):
 __pycache__
 # End of gitignore""")
 
+        os.makedirs("another_folder", exist_ok=True)
+        os.chdir("another_folder")
+        with open('.gitignore', 'w') as f:
+            f.write("""
+# Start of gitignore
+*.js
+# End of gitignore""")
+        os.chdir(os.path.dirname(os.path.realpath(__file__)))
+        os.chdir("file_globs_gitignore_testfiles")
+        os.makedirs("data", exist_ok=True)
+        os.chdir("data")
+        os.makedirs("sample", exist_ok=True)
+        os.chdir("sample")
+        with open('.gitignore', 'w') as f:
+            f.write("""
+# Start of gitignore
+*.html
+# End of gitignore""")
+        os.chdir(os.path.dirname(os.path.realpath(__file__)))
+        os.chdir("file_globs_gitignore_testfiles")
         files = [os.path.join("src", "main.c"),
             os.path.join("src", "main.h"),
             os.path.join("src", "lib", "ssl.c"),
             os.path.join("src", "tests", "main.c"),
             os.path.join("src", "main.py"),
             os.path.join("src", "upload.c"),
             os.path.join("another_folder", "another_file.c"),
-            os.path.join("data", "sample","index.css"),
+            os.path.join("data", "sample", "index.css"),
```

Comment:
Irrelevant change, we have an issue for this https://github.com/coala/coala-quickstart/issues/113 , so if you want to do this change, please do all of the changes in separate commit, or better, separate PR :)